R/summary_functions.R

```diff
@@ -191,6 +269,44 @@ print.coverages <- function(x, ...) {
   invisible(x)
 }
 
+
+
+#' Print the branch coverage of a coverage object
+#'
+#' @param x the coverage object to be printed
+#' @param group whether to group coverage by filename or function
+#' @param ... additional arguments ignored
+#' @return The coverage object (invisibly).
+#' @export
+print.branch_coverage <- function(x, group = c("filename", "functions")) {
+    stopifnot(inherits(x, "coverage"))
+
+    df_br <- tally_branch_coverage(x)
+    ## empty data.frame means no branch was found
```

Comment:
This comment does not bring any additional information. Comments should not become code subtitles. The code should explain as much as it can by being well written. Comments should provide a more context description, rationale, motivation, extra explication.